.circleci/config.yml

```diff
@@ -94,40 +94,57 @@ jobs:
             - ./dev-dbs-clean-for-these-branches.txt
           key: dev-dbs-clean-for-these-branches--{{ checksum "./active-branches.txt" }}-v2
 
+  lighthouse:
+    working_directory: "~/InfoBase"
+    docker:
+      - image: markhobson/node-chrome:latest
+        <<: *dockerhub_auth
+    steps:
+      - checkout:
+          path: "~/InfoBase"
+      - run: ./scripts/ci_scripts/setup_lighthouse_env.sh
+      - run: ./scripts/ci_scripts/cleanup_lighthouse_db.sh
+      - run:
+          name: lhci autorun
+          command: node ./node_modules/@lhci/cli/src/cli.js autorun --config ./scripts/ci_scripts/lighthouserc.js
```

Comment:
Was bringing this back this (or adding it?) intentional? I believe we dropped the lighthouse job on master quite a while ago.